.changes/next-release/Bug Fix-2ce4c789-d0e5-425c-affb-6cac6e5069fd.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Improve CodePercentage telemetry reporting"
```

When reading the changelog, customers will wonder which feature this is referring to.
```suggestion
	"description": "CodeWhisperer: Improve CodePercentage telemetry reporting"
```